src/main/scala/wdl4s/WdlSyntaxErrorFormatter.scala

```diff
@@ -45,6 +45,14 @@ case class WdlSyntaxErrorFormatter(terminalMap: Map[Terminal, WdlSource]) extend
      """.stripMargin
   }
 
+  def missingListItems(method: String, required: Int, found: Int, last: Terminal): String = {
+    s"ERROR: $method requires $required items, but only found $found"
+  }
+
+  def missingTerminator(method: String, terminal: TerminalIdentifier, last: Terminal): String = {
+    s"ERROR: $method requires a terminator after each element"
+  }
+
```

Comment:
I kind of cut corners with these error messages because they will not happen with WDL source. Right now, the parser will pass in `null` in for the value of `last`, but that will be fixed once I engineer some better error handling in Hermes